### PR TITLE
Remove gutenberg

### PR DESCRIPTION
**Pull request by nmboavida**

### CODE REVIEW

**Comment by nmboavida** [comment]
I commented out plenty of LOCs in Gutenberg, since we may reintroduce some of that logic, likely in a different form, whenever we settle on a more straight forward `mint-to-launchpad` flow.

**Review by Suficio — APPROVED** [review]
Is Gutenberg still Gutenberg once it has been commented out of existence and uncommented back in in its entirety?

File: gutenberg/src/schema.rs [hunk 1]
```diff
@@ -92,47 +86,44 @@ impl Schema {
...
-        vars.insert("market_type", &market_type);
-        vars.insert("market_module", market_module);
-        vars.insert("market_module_imports", &market_module_imports);
-        vars.insert("slingshot_import", &slingshot_import);
-        vars.insert("is_embedded", &is_embedded_str);
-        vars.insert("mint_function", &mint_func);
-        vars.insert("define_market_arguments", &define_market_arguments);
-        vars.insert("market_arguments", &market_arguments);
+
+        // TODO: This is commented out but may be added back in future
```
Comment (bausano) [inline gutenberg/src/schema.rs:116]: can we perhaps delete it then? I have bad experience with code bases with commented out code. Or if you'd prefer to leave it in, some explanation on what's this and what needs to be done to be uncommented.

File: gutenberg/src/schema.rs [hunk 2]
```diff
@@ -154,72 +145,76 @@ impl Schema {
...
-    pub fn write_market_arguments(&self) -> Box<str> {
-        match self.launchpad.market_type {
-            MarketType::FixedPrice { .. } => "whitelist, prices,",
-            MarketType::Auction { .. } => "whitelist, reserve_prices,",
-        }
-        .into()
-    }
+    // /// Associated function that generates Move code to declare and push market
+    // /// specific data.
+    // pub fn write_define_market_arguments(&self) -> Box<str> {
```
Comment (bausano) [inline gutenberg/src/schema.rs:167]: TODO

File: gutenberg/src/types.rs [hunk 3]
```diff
@@ -9,129 +9,178 @@ use serde::Deserialize;
 #[derive(Debug, Deserialize)]
 pub enum NftType {
     // TODO: Need to add support for Soulbound
-    Unique,
+    Classic,
+    // TODO: To be added back
```
Comment (bausano) [inline gutenberg/src/types.rs:13]: TODO

File: gutenberg/src/types.rs [hunk 4]
```diff
@@ -9,129 +9,178 @@ use serde::Deserialize;
...
+                        attribute_values,
+                        ctx,
+                    );
+
+                    slot::add_nft(slot, market_id, nft, ctx);
                 }}",
                 witness = witness,
-                market_type = market_type,
             ),
+            // NftType::Collectible => format!(
```
Comment (bausano) [inline gutenberg/src/types.rs:126]: TODO

File: gutenberg/tests/generate.rs [hunk 5]
```diff
@@ -17,28 +17,28 @@ fn example_schema() {
...
     let expected = fs::read_to_string("../examples/suimarines.move").unwrap();
 
     assert_equal(config, expected);
 }
 
-#[test]
-fn collectible() {
-    let config = File::open("./examples/collectibles.yaml").unwrap();
-    let expected = fs::read_to_string("../examples/collectibles.move").unwrap();
+// #[test]
```
Comment (bausano) [inline gutenberg/tests/generate.rs:27]: TODO

**Pull request merged**